src/Config.php

```diff
@@ -3312,7 +3312,7 @@ public function showFormLogs()
 
         echo "<tr class='tab_bg_1'><th colspan='4'>" . _n('Software', 'Software', Session::getPluralNumber()) . "</th></tr>";
         echo "<tr class='tab_bg_1'><td class='center'>" .
-           __("Installation/uninstallation of software on items") . "</td><td>";
+           __("Installation/update/uninstallation of software on items") . "</td><td>";
```

Comment:
Is this really necessary?

Reply:
Turns out, it's not really that necessary after all.